types/appconfig.pp

```diff
@@ -0,0 +1,24 @@
+type Puppetwebhook::Config = Struct[{
+  protected                 => Boolean,
+  user                      => String,
```

Comment:
Should we also enforce the minimal string length? I think `String[1]` is a good approach in almost all locations.